templates/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "<% .Name %>",
+  "version": "1.0.0",
+  "description": "A simple http server: hello world",
+  "main": "app.js",
+  "scripts": {
+    "start": "node .",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Having this action here might be a bit misleading because we're not using it in the ci pipeline?